SECURITY.md

```diff
@@ -181,6 +189,13 @@ the community they pose.
   no scenario that requires modification of that configuration file is
   considered a vulnerability.
 
+#### Uncontrolled Resource Consumption (CWE-400) on outbound connections
+
+* If Node.js is asked to connect to a remote site and return an
+  artifact, it is not considered a vulnerability if the size of
+  that artifact is large enough to impact performance and or
+  cause the runtime to run out of resources.
```

Comment:
Is this actually true? Say I host a malicious server that triggers a hash table collision denial-of-service in node's http client - that's considered a vulnerability, right?

Reply:
@bnoordhuis I think your example is different in that it does not relate to the size of the artifact returned.